frame/staking/src/lib.rs

```diff
@@ -1514,6 +1598,18 @@ pub mod pallet {
 					ledger.active = Zero::zero();
 				}
 
+				let min_active_bond = if Nominators::<T>::contains_key(&ledger.stash) {
+					MinNominatorBond::<T>::get()
+				} else if Validators::<T>::contains_key(&ledger.stash) {
+					MinValidatorBond::<T>::get()
+				} else {
+					Zero::zero()
+				};
+
+				// Make sure that the user maintains enough active bond for their role.
+				// If a user runs into this error, they should chill first.
```

Comment:
I would consider a mirroring note for this error in the doc comments for this extrinsic